keras/utils/layer_utils.py

```diff
@@ -24,6 +24,15 @@ def layer_from_config(config, custom_objects=None):
     if custom_objects:
         get_custom_objects().update(custom_objects)
 
+    # New behaviour
+    if custom_objects and 'layer_class' in custom_objects:
+        layer_class = custom_objects['layer_class']
+        # Remove layer class from custom_objects as it's not needed anymore
+        custom_objects.pop('layer_class', None)
+        for cls_key in custom_objects:
+            globals()[cls_key] = custom_objects[cls_key]
```

Comment:
Tried to figure this out but wasn't 100% clear to me... why do we need to dump the `custom_objects` in the global variable space here?